src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newData = date.split(fromFormat[fromFormat.length - 1]);
+  const fromFormatDate = {};
+  const toFormatDate = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    fromFormatDate[fromFormat[i]] = newData[i];
+  }
+
+  if (fromFormatDate.hasOwnProperty('YYYY')) {
+    fromFormatDate.YY = fromFormatDate.YYYY.slice(2);
+  }
+
+  if (fromFormatDate.YY >= 30) {
+    fromFormatDate.YYYY = '19' + fromFormatDate.YY;
```

Comment:
You can rewrite like this
```suggestion
    fromFormatDate.YYYY = `19${fromFormatDate.YY}`;
```